src/cmd/build.rs

```diff
@@ -493,30 +493,37 @@ fn check_wasm_opt_version_compatibility(wasm_opt_path: &Path) -> Result<()> {
     let version_stdout = str::from_utf8(&cmd.stdout)
         .expect("Cannot convert stdout output of wasm-opt to string")
         .trim();
-    let re = Regex::new(r"wasm-opt version (\d+)\s+").unwrap();
+    let re = Regex::new(r"wasm-opt version (\d+)").unwrap();
```

Comment:
I didn't spot it in the original PR but this should be an `expect("invalid regex")` or such.

Reply:
Argh! Maybe we should consider adding a clippy rule for `unwrap()`.